src/android/Capture.java

```diff
@@ -266,7 +278,18 @@ private void captureAudio(Request req) {
 
         try {
             Intent intent = new Intent(android.provider.MediaStore.Audio.Media.RECORD_SOUND_ACTION);
-            this.cordova.startActivityForResult((CordovaPlugin) this, intent, req.requestCode);
+
+            String timeStamp = new SimpleDateFormat("yyyyMMddHHmmssSSS").format(new Date());
+              String fileName = "AUDIO_" + timeStamp + ".wav";
```

Comment:
just a suggestion, I would extract this way of getting a timestamp to a function, something like
```java
private static final String DATE_FORMAT = "yyyyMMddHHmmssSSS";
private File getFile(String type, string extension) {
  String timeStamp = new SimpleDateFormat(DATE_FORMAT).format(new Date());
  String filename = String.format("$s_%s.%s",type, timeStamp, extension);
  return new File(getTempDirectoryPath(), filename);
}
```

Reply:
Good call, updated!